apps/src/templates/progress/ProgressLevelSet.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { PropTypes } from 'react';
+import FontAwesome from '../FontAwesome';
+import ProgressBubbleSet from './ProgressBubbleSet';
+import color from '@cdo/apps/util/color';
+import i18n from '@cdo/locale';
+import { levelType } from './progressTypes';
+
+import { BUBBLE_COLORS } from '@cdo/apps/code-studio/components/progress/progress_dot';
+
+const styles = {
+  table: {
+    marginTop: 12
+  },
+  stepButton: {
+    // TODO - might run into problems with i18n
```

Comment:
Yeah, ideally this wouldn't be hard-coded, but we want all the buttons to be the same size. I'm going to leave this TODO in for now, but at some point would like to make the size not hard-coded.